tests/lint/unittest_lint.py

```diff
@@ -311,7 +311,7 @@ def test_enable_message_block(initialized_linter: PyLinter) -> None:
     # meth6
     assert not linter.is_message_enabled("E1101", 57)
     assert linter.is_message_enabled("E1101", 61)
-    assert not linter.is_message_enabled("E1101", 64)
+    assert linter.is_message_enabled("E1101", 64)
```

Comment:
Breaking change to warn about in changelog ?

Reply:
[a95880d](https://github.com/pylint-dev/pylint/pull/7767/commits/a95880d6d33b7adea0834b563f5298bf5c779bdf)